Packs/CortexXDR/Playbooks/playbook-Cortex_XDR_-_Cloud_IAM_User_Access_Investigation.yml

```diff
@@ -0,0 +1,915 @@
+  "7":
+    id: "7"
+    taskid: b17705e9-b1ad-475d-87e6-c0974d5fc8fa
+    type: condition
+    task:
+      id: b17705e9-b1ad-475d-87e6-c0974d5fc8fa
+      version: -1
+      name: Decision making - true/false-positive alert
+      description: Based on the collected data investigation, is it a true positive event?
+      type: condition
+      iscommand: false
+      brand: ""
+    nexttasks:
+      "No":
+      - "10"
+      "Yes":
+      - "8"
+    separatecontext: false
+    continueonerrortype: ""
+    view: |-
+      {
+        "position": {
+          "x": 670,
+          "y": 1020
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    message:
+      to:
+        simple: Analyst
+      subject:
+        simple: Based on the collected data investigation, is it a true positive event?
+      body:
+        simple: Based on the collected data investigation, is it a true positive event?
+      methods:
+      - email
+      format: ""
+      bcc:
+      cc:
+      timings:
+        retriescount: 2
+        retriesinterval: 360
+        completeafterreplies: 1
+        completeafterv2: true
+        completeaftersla: false
+      replyOptions:
+      - "Yes"
+      - "No"
+    skipunavailable: false
+    quietmode: 2
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "8":
+    id: "8"
+    taskid: 4b87a31c-f367-4ab6-8ac7-3f9a2f3d179a
+    type: title
+    task:
+      id: 4b87a31c-f367-4ab6-8ac7-3f9a2f3d179a
+      version: -1
+      name: Remediation
+      type: title
+      iscommand: false
+      brand: ""
+      description: ''
+    nexttasks:
+      '#none#':
+      - "9"
+    separatecontext: false
+    continueonerrortype: ""
+    view: |-
+      {
+        "position": {
+          "x": 450,
+          "y": 1220
+        }
+      }
+    note: false
+    timertriggers:
+    - fieldname: remediationsla
+      action: start
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "9":
+    id: "9"
+    taskid: 89e1b907-b4c4-4dfa-8579-17969e71e1a2
+    type: playbook
+    task:
+      id: 89e1b907-b4c4-4dfa-8579-17969e71e1a2
+      version: -1
+      name: Cloud Response - Generic
+      description: |-
+        This playbook provides response playbooks for:
+        - AWS
+        - Azure
+        - GCP
+
+        The response actions available are:
+        - Terminate/Shut down/Power off an instance
+        - Delete/Disable a user
+        - Delete/Revoke/Disable credentials
+        - Block indicators
+      playbookName: Cloud Response - Generic
+      type: playbook
+      iscommand: false
+      brand: ""
+    nexttasks:
+      '#none#':
+      - "17"
+    scriptarguments:
+      AWS-accessKeyRemediationType:
+        complex:
+          root: inputs.AWS-accessKeyRemediationType
+      AWS-resourceRemediationType:
+        complex:
+          root: inputs.AWS-resourceRemediationType
```

Comment:
I think for those types of alerts - we should only remediate the user (the resource can be your management server) which we don't want to block.
So we should block the user and the source IP

Reply:
let's remove all inputs that not relate to the user

Reply:
Agreed. Done